articles/android-rxjava/index.md

```diff
@@ -0,0 +1,127 @@
+#### RxJava in Android
+ReactiveX has created libraries for almost all programming languages. It has also extended this to frameworks like Android. This has helped bring the functional reactive paradigm to the languages and frameworks we use. In Android, it helps bring efficiency in application performance. It also ensures a clean and maintainable code. This article will go through the basics of RxJava and how to use it in Android development.
+
+#### Prerequisites
+To follow through this article, you need :
+* Android Studio installed
+* Basic understanding of Android development using Kotlin
+* Experience with retrofit and room
+
+#### What we'll look at :
+1. [Observables](#Observables)
+    * [observable](#observable)
+    * [flowable](#flowable)
+    * [single](#single)
+    * [completable](#completable)
+
+2. [Schedulers](#Schedulers)
+    * [newThread](#newThread)
+    * [io](#io)
+    * [computation](#computation)
+    * [mainThread](#mainThread)
+
+3. [Application in Android](#Application-in-Android)
+    * [retrofit](#retrofit)
+    * [room](#room)
+
+
+#### Observables
+RxJava, like the other ReactiveX libraries, uses the observable pattern. This pattern involves an `observer` who subscribes to an `observable`. The observer then receives data emitted by the observable and performs actions on it. An observable is an object or entity that emits data. We have five types of observers in RxJava, but we will look at only four. This is because the `Maybe` observable is not encouraged in Android applications. These observables have some standard methods in their classes to help in managing the emissions. The methods are `onNext`, `onCompleted`, `onSuccess` and `onError`.
```

Comment:
"like other ReactiveX libraries" flows better. Omit "the".
"We have five types of observers..." I believe you meant *observables*.
"... to help manage emissions" flows well on the second last line.

Reply:
done...